acceptance/tests/facts/networking_facts.rb

```diff
@@ -0,0 +1,80 @@
+test_name 'FACT-1361 - C59029 networking facts should be fully populated'
```

Comment:
test_name and step should have `do/end` to create blocks, and proper indentation in logs